applications/matter_bridge/src/app_task.cpp

```diff
@@ -48,6 +48,42 @@ static bt_uuid *sUuidLbs = BT_UUID_LBS;
 static bt_uuid *sUuidEs = BT_UUID_ESS;
 static bt_uuid *sUuidServices[] = { sUuidLbs, sUuidEs };
 static constexpr uint8_t kUuidServicesNumber = ARRAY_SIZE(sUuidServices);
+/**
+ * @brief Blink rates for indication the BLE Connectivity Manager state.
+ *
+ */
+constexpr static uint32_t kPairingBlinkRate{ 100 };
+constexpr static uint32_t kScanningBlinkRate_ms{ 300 };
+constexpr static uint32_t kLostBlinkRate_ms{ 1000 };
+
+void BLEStateChangeCallback(Nrf::BLEConnectivityManager::State state)
```

Comment:
nit: Shouldn't be part of AppTask class? Or at least be a static function.

Reply:
It's already "static" since it is in the anonymous namespace.